packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/property-table/cells/value-cell.tsx

```diff
@@ -41,9 +47,15 @@ export const renderValueCell: CustomRenderer<ValueCell> = {
         iconColor: "#91A5BA",
         iconSize: 16,
       });
+    } else {
+      // draw plain text
+      ctx.fillText(String(value), left, yCenter);
     }
 
-    ctx.fillText(String(value), left, yCenter);
+    const tooltipManager = new GridTooltipManager(args);
+    const tooltipInteractables = tooltipManager.drawAndCreateInteractables();
+
+    InteractableManager.setInteractablesForCell(args, tooltipInteractables);
```

Comment:
Could this all be combined somehow? Are there circumstances in which people want to create a manager without drawing interactables, or drawing interactables without setting them for a cell?

Reply:
`InteractableManager.setInteractablesForCell` should be called `just once` for each cell's draw method. So if we possibly want to use any other interactable on top of `GridTooltipManager`, we should be using it like this. That's why `drawAndCreateInteractables` returns the interactables array. So we can set interactables at the end of the draw method.

Reply:
See [next review](https://github.com/hashintel/hash/pull/1386#pullrequestreview-1180397155) – maybe `GridTooltipManager` could just be a function like 
```
const tooltipInteractables = createRightTooltips(args);

InteractableManager.setInteractablesForCell(args, tooltipInteractables);
```

Since it's just for drawing tooltips to the right of property cells, I think calling it a tooltip manager is going to be confusing .